main.py

```diff
@@ -1,10 +1,45 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as players_file:
+
+        players = json.load(players_file)
+
+    for player_name, player_info in players.items():
+
+        guild = None
+        guild_dict = player_info["guild"]
+        if isinstance(guild_dict, dict):
+            guild = Guild.objects.get_or_create(
+                name=guild_dict["name"],
+                description=guild_dict["description"]
+            )[0]
+
+        race_dict = player_info["race"]
+        race = Race.objects.get_or_create(
+            name=race_dict["name"],
+            description=race_dict["description"]
+        )[0]
```

Comment:
it looks a little better
```suggestion
        race, _ = Race.objects.get_or_create(
            name=race_dict["name"],
            description=race_dict["description"]
        )
```